src/index.js

```diff
@@ -1,7 +1,48 @@
+const client = require('prom-client');
 const config = require('config');
 const fetchStatuCakeData = require('./statuscake');
 const server = require('./server');
+const Storage = require('./statuscakedata');
 
-this.interval = setInterval(fetchStatuCakeData, config.statuscake.interval);
+const regex = /\s?([-])\s?/;
+
+const getTagsFromTitle = Title => Title.split(regex);
+
+const g = new client.Gauge({
+  name: 'metric_loadtime',
```

Comment:
To respect ou prometheus conventions (available in Confluence), it should be something like : `frontend_pagespeed_loadtime_seconds` 🍰 
And it should be converted in seconds https://prometheus.io/docs/practices/naming/#base-units

Reply:
Thanks.